test/NHSD.BuyingCatalogue.Documents.API.Tests/Repository/AzureBlobDocumentRepositoryTests.cs

```diff
@@ -21,7 +21,7 @@ namespace NHSD.BuyingCatalogue.Documents.API.UnitTests.Repository
     internal sealed class AzureBlobDocumentRepositoryTests
     {
         [Test]
-        public async Task DocumentNameDownloadAsync_ReturnsBlobDownloadInfo()
+        public async Task DownloadAsync_string_ReturnsBlobDownloadInfo()
```

Comment:
I think the lowercase `string` looks a bit odd although I understand why you chose the lowercase option – would you mind changing it to upper case here and elsewhere in the file?

Reply:
👍